platforms/jvm/scala/src/main/java/org/gradle/api/plugins/scala/ScalaBasePlugin.java

```diff
@@ -172,7 +191,23 @@ private void configureConfigurations(final ProjectInternal project, Category inc
         });
     }
 
-    private void configureSourceSetDefaults(final ProjectInternal project, Category incrementalAnalysisCategory, final Usage incrementalAnalysisUsage) {
+    private Provider<ResolvableConfiguration> getCompilerRuntimeClasspath(Project project, ScalaPluginExtension scalaPluginExtension) {
+        Provider<DependencyScopeConfiguration> scalaCompiler = project.getConfigurations().dependencyScope("scalaCompiler", conf -> {
```

Comment:
Maybe instead of "scalaCompiler", this is actually the "scalaToolchain" because it's used for compilation and ScalaDoc

Reply:
Renamed *compiler* to *toolchain*